pages/price-feeds/create-your-first-pyth-app/evm/part-1.mdx

```diff
@@ -16,31 +16,33 @@ This tutorial is divided into two parts: \
 
 ## Create a contract and fetch prices from Pyth oracles
 
-In this part of the tutorial, we will create a contract that reads price from Pyth and uses it to calculate the amount of ETH required to mint an NFT.
+In this part of the tutorial, we will create a contract that reads the price from Pyth and uses it to calculate the amount of ETH required to mint an NFT.
 After that, we will write tests to ensure that the contract works as expected.
 
 ### Preliminaries
 
 This tutorial uses Foundry to perform the contract development tasks.
-Please follow the [Foundry installation instructions](https://book.getfoundry.sh/getting-started/installation) if you do not already have it installed.
-
-Part of this tutorial also uses the command line tools `curl` and `jq`.
 Please make sure these are installed on your system before continuing.
 
+- [foundry](https://book.getfoundry.sh/getting-started/installation)
+- [node](https://nodejs.org/en/download/)
+- [curl](https://curl.se/download.html)
+- [jq](https://stedolan.github.io/jq/download/)
+
 ### Create a Foundry project
 
 Create a new directory to hold your app and a `contracts` directory within.
 
 ```bash copy
 mkdir my_first_pyth_app
-mkdir my_first_pyth_app/contracts
-cd my_first_pyth_app/contracts
+mkdir my_first_pyth_app/contracts && cd my_first_pyth_app/contracts
```

Comment:
nitpick: If you're going to do this in one block of commands (which is fine), then the block above describing the command to run also should describe what `forge init` does